# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../qcad/compile.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- qcad-2.0.5.0-1-community.src/qcadlib/src/engine/rs_layer.cpp.orig	2006-01-25 23:59:06.000000000 +0100
+++ qcad-2.0.5.0-1-community.src/qcadlib/src/engine/rs_layer.cpp	2006-01-25 23:59:47.000000000 +0100
@@ -55,7 +55,7 @@
     os << " name: " << l.getName().latin1()
     << " pen: " << l.getPen()
 	<< " frozen: " << (int)l.isFrozen()
-	<< " address: " << (int)(&l)
+	<< " address: " << (long)(&l)
     << std::endl;
     return os;
 }
--- qcad-2.0.5.0-1-community.src/qcadlib/src/engine/rs_entity.cpp.orig	2006-01-25 23:58:00.000000000 +0100
+++ qcad-2.0.5.0-1-community.src/qcadlib/src/engine/rs_entity.cpp	2006-01-25 23:58:13.000000000 +0100
@@ -861,7 +861,7 @@
         os << " layer: NULL ";
     } else {
         os << " layer: " << e.layer->getName().latin1() << " ";
-        os << " layer address: " << (int)(e.layer) << " ";
+        os << " layer address: " << (long)(e.layer) << " ";
     }
 
     os << e.pen << "\n";
